### ISSUE TITLE
Table with fixed header has incorrect y-scrollbar size

### ISSUE THREAD

**Issue by asimonok** [henrybuilt/react-sticky-table#46]
Hi,

When we have fixed header, vertical scrollbar has incorrect size. Because its height without subtraction height fixed header.
Please see attachment for more details.

I hope you will fix it at the nearest time
Thanks, Alex!

![2018-08-23 9 23 55](https://user-images.githubusercontent.com/15867703/44508367-b2462400-a6b6-11e8-9f38-a6f6eb2f2882.png)

**Comment by dominikdosoudil** [comment]
I think that the problem is here:

src/index.js line 269
```
    var height = this.dom.bodyTable.offsetHeight + this.dom.stickyHeader.offsetHeight;
    this.dom.yScrollbar.firstChild.style.height = height + 'px';
```

The height of the scrollbar should be same as the table body that isn't sticky. Means adding this.dom.stickyHeader.offsetHeight makes sense only in case header isn't sticky.

However wouldn't that condition break another use case?

**Comment by maxhudson** [comment]
resolved in v3.0.1
